automated-comments/scala/general/use_string_interpolate.md

```diff
@@ -0,0 +1,3 @@
+Use [string interpolation](https://docs.scala-lang.org/overviews/core/string-interpolation.html) to dynamically build a string, rather than using string concatenation or StringBuilder. 
```

Comment:
```suggestion
Use [string interpolation](https://docs.scala-lang.org/overviews/core/string-interpolation.html) to dynamically build a string, rather than using string concatenation or `StringBuilder`. 
```